oak_containers_syslogd/src/log_relay.rs

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use std::collections::HashMap;
+
+use anyhow::Result;
+use oak_containers_orchestrator_client::LauncherClient;
+use systemd::journal;
+
+pub async fn run(launcher_client: LauncherClient) -> Result<()> {
+    let mut journal = journal::OpenOptions::default()
+        .all_namespaces(true)
+        .open()?;
+    journal.seek_head()?;
+
+    loop {
+        // Read all entries we can.
+        let mut entries = Vec::new();
+        while let Some(entry) = journal.next_entry()? {
```

Comment:
Should there be some cutoff so that the vector does not grow too large before we log?